src/client/rpc.rs

```diff
@@ -291,6 +302,11 @@ impl RpcClient {
             .set_code()?;
 
         if res.is_invalid() {
+            counter!(format!(
+                "rpc_new_payload_v4_invalid_payload_{}",
```

Comment:
many of the metrics can be simplified by having the method name and payload source as an attribute